bin/destroy.sh

```diff
@@ -81,6 +81,11 @@ if pulumi config get kubernetes:infra_type -C ${script_dir}/../pulumi/python/con
     sleep 5
     ${script_dir}/destroy_kube.sh
     exit 0
+  elif [ $INFRA == 'DO' ]; then
```

Comment:
If we need per provider logic, it may be better to have a fixed path where we look for a script with a filename that includes an identifier. If the script exists, then we execute it. If we start to fork a lot of logical paths for each provider, we may quickly have a mess. Take this with a grain of salt.

Reply:
Once we get #81 and #82 sorted, I foresee this all being collapsed into one script where the provider is passed in as an argument (or read from the config).